VERSIONLOG.md

```diff
@@ -1,5 +1,8 @@
 # TACA Version Log
 
+##20230210.1
+Update 20230117.1 after local troubleshooting.
```

Comment:
I think I would prefer a more informative message here, what and/or why was something updated?

Reply:
Oh, and also maybe increase the version in https://github.com/SciLifeLab/TACA/blob/master/taca/__init__.py 🙏

Reply:
Done, see [5cbcf66](https://github.com/SciLifeLab/TACA/pull/350/commits/5cbcf66fcea3d992cba7b6e25a35257443b3d26e)